Fixed highlight (#3327)
ezsystems/developer-documentation: docs/content_management/rich_text/create_custom_richtext_block.md

Then, create a subscriber that converts a string of data into XML code.
Create a `src/Event/Subscriber/RichTextBlockSubscriber.php` file.

In line 28, `my_block` is the same name of the block that you defined in line 3 above.
Line 28 also implements the `PreRender` method.

> Lines 41-51 handle the conversion of content into an XML string:

Author: ```suggestion
Lines 37-47 handle the conversion of content into an XML string:
```

> Line 28 also implements the `PreRender` method.

Reviewer: This "implements the `PreRender` method" is weird because this isn't what is happening at this line 28.

Reviewer: ```suggestion
Line 28 links this block `PreRenderEvent` to a method using `BlockRenderEvents::getBlockPreRenderEventName()`.
```

> ``` php hl_lines="28 41-51"

Author: ```suggestion
``` php hl_lines="28 37-47"
```
